src/pages/patientView/structuralVariant/PatientViewStructuralVariantTable.tsx

```diff
@@ -479,7 +479,21 @@ export default class PatientViewStructuralVariantTable extends React.Component<
             ) {
                 return (
                     <div className="alert alert-info" role="alert">
-                        Structural Variants are not available.
+                        Study has no Structual Variant Data.
+                    </div>
+                );
+            } else if (
+                !this.props.store.genePanelDataByMolecularProfileIdAndSampleId
```

Comment:
lets move this ugly condition to the top of the render method and assign it to a nice local variable 
```
const isProfiled = ...
```